crates/autopilot/src/infra/persistence/fee_policy/mod.rs

```diff
@@ -0,0 +1,41 @@
+use {
+    crate::{driver_model::solve::FeePolicy, protocol::fee},
+    anyhow::Context,
+    database::byte_array::ByteArray,
+    model::order::OrderUid,
+};
+
+impl super::Persistence {
+    pub async fn store_fee_policies(
+        &self,
+        auction_id: model::auction::AuctionId,
+        order_uids: impl Iterator<Item = OrderUid>,
+        fee_policies: &fee::Policies,
+    ) -> anyhow::Result<()> {
+        let mut ex = self.postgres.clone().pool.begin().await.context("begin")?;
+        for order_uid in order_uids {
+            for fee_policy in fee_policies.get(&order_uid).unwrap_or_default() {
+                let fee_policy_dto = database::fee_policies::FeePolicy {
```

Comment:
If this is the dto for storing in the db, do we then need an extra `FeePolicyRow` struct inside the postgres impl? Or should the dto be what we can right straight into the db?

Reply:
The `database` crate should not have any dependencies, so the `FeePolicyRow` should remain. Do I miss anything?

Reply:
The FeePolicy -> FeePolicyRow conversion contains some internal logic that should not be visible through the DTO. But we can remove the additional layer for sure.